<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1335</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="mainLayout" stretch="1,6">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="userInputs">
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="addCity">
            <property name="text">
             <string>Add City</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addStreetButton">
            <property name="text">
             <string>Add Street</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dijkstraButton">
            <property name="text">
             <string>Find Connection</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="loadMapButton">
            <property name="text">
             <string>Load Map</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="debugInputs" native="true">
          <layout class="QVBoxLayout" name="debugInputsLayout">
           <item>
            <widget class="QLabel" name="label_eingabe">
             <property name="text">
              <string>Ihre Eingabe</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_test">
             <property name="placeholderText">
              <string>Ihre Eingabe</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_test">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>WhaAaaAAt??</string>
             </property>
             <property name="text">
              <string>Run Test!?</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mapButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Test Map</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="streetButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Test simple Street</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cityButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Test Draw City</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mapStreetButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Test Map Street</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="abstractMapButton">
             <property name="text">
              <string>Test Abstract Map</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="testDijkstraButton">
             <property name="text">
              <string>Test Dijkstra</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fillMapButton">
             <property name="text">
              <string>Fill Map</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="debugBox">
          <property name="text">
           <string>Debug</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1335</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionTrenner"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuScene">
    <property name="title">
     <string>Scene</string>
    </property>
    <addaction name="actionClear_Scene"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuScene"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionTrenner">
   <property name="text">
    <string>Trenner hinzuf√ºgen</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionClear_Scene">
   <property name="text">
    <string>Clear Scene</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
